docs/pages/distribution/publishing-websites.mdx

```diff
@@ -354,19 +347,19 @@ For example, if the GitHub repository is `expo-gh-pages`, the following will be
 
 </Step>
 
-<Step label="5">
+<Step label="4">
 
 Because Expo uses underscores in generated files, you need to disable jekyll by creating a **.nojekyll** file in the **public** directory.
 
 <Terminal cmd={['$ mkdir public && touch public/.nojekyll']} />
 
-This file will be copied to the **dist** directory when you build your project.
+Files in the **public** directory will be copied to the **dist** directory when you build your project.
 
-> You also need to run `gh-pages` with the `-t` flag to ensure files starting with `.` are included in the deployment.
+> You also need to run `gh-pages` with the `-t` flag to ensure files starting with `.` are included in the deployment, which is handled in the scripts added to **package.json** in the next step.
```

Comment:
```suggestion
```

Let's remove this. The next step is something not optional for the user and they have to follow it in order to publish their web app.

Instead, this context should be in step 5 after running the command.